src/components/cardbuilder/cardBuilder.js

```diff
@@ -1336,9 +1336,9 @@ import ServerConnections from '../ServerConnections';
                 cardImageContainerClose = '</div>';
             } else {
                 // Don't use the IMG tag with safari because it puts a white border around it
-                cardImageContainerOpen = imgUrl ? ('<button data-action="' + action + '" class="' + cardImageContainerClass + ' ' + cardContentClass + ' itemAction lazy" data-src="' + imgUrl + '" ' + blurhashAttrib + '>') : ('<button data-action="' + action + '" class="' + cardImageContainerClass + ' ' + cardContentClass + ' itemAction">');
+                cardImageContainerOpen = imgUrl ? ('<div data-action="' + action + '" class="' + cardImageContainerClass + ' ' + cardContentClass + ' itemAction lazy" data-src="' + imgUrl + '" ' + blurhashAttrib + '>') : ('<div data-action="' + action + '" class="' + cardImageContainerClass + ' ' + cardContentClass + ' itemAction">');
```

Comment:
Why change this? A button feels more semantically correct for an element with an action attached to it on click.

Reply:
@thornbill At first, I thought it was caused because it was a button element, later I realized your changes in image loader. I left this as the div per-se it's not the card element or the play button, it's the image container, so i think it makes sense that it stays as a div instead. The ``cardOverlayContainer`` element will already handle the click action.

With this, this whole if/else can be refactored to a single line now that we don't distinguish between elements, but I didn't do that because I don't think we should change that in a backport, just in case bad stuff happens.

Reply:
My main concern would be if this has unintended side effects for keyboard navigation or screen readers. The card itself often has an action defined for it in addition to the buttons that are overlayed.

Reply:
@thornbill Good point, and, as I mentioned before, better to not touch unrelated things because we can know what can happens, so better not to backport this stuff :)

Force pushed without those modifications.